chapters/03-kubernetes-deployment.md

```diff
@@ -35,23 +40,39 @@ NAME                             DESIRED   CURRENT   READY     AGE
 replicaset.apps/web-65899c769f   1         1         1         11s
 ```
 
+_"I just wanted a container! Why do I get three different objects?"_
+
 Instead of getting a container, you got a whole zoo of unknown beasts:
 
 - a *deployment* (called `web` in this example),
 - a *replicaset* (`web-65899c769f`),
 - a *pod* (`web-65899c769f-dhtdx`).
 
-Note: you can ignore the *service* named `kubernetes` in the example
-above; that one already existed before the `kubectl run` command.
+Note: you can ignore the *service* named `kubernetes` in the example above; that one already existed before the `kubectl run` command.
 
-_"I just wanted a container! Why do I get three different objects?"_
+**Kubernetes Containers in Versions 1.8 and Higher**
```

Comment:
1.18

Reply:
I did not test the quick demo section with version 1.18. I'll test it now and let you know.